### DISCUSSION TITLE
Retrieve the signed URLs from all files in the bucket.

### DISCUSSION THREAD

**Discussion by rgroothuis** [supabase/supabase#13468] [Questions] [answered]
Currently I've two Supabase calls, one for getting all the file names in a folder located in a bucket. The next call is the one to get all the signed URL's of those files. I was just wondering if there is a method to get all signed URLs of all the files in one folder in a bucket? This will save one Supabase call and make the App a bit faster. Thanks.

**Comment by GaryAustin1** [comment, marked as answer]
getSignedUrls only takes an array of the filenames.  No wildcards.

**Reply by GaryAustin1** [reply]
You should generate a feature request here:  https://github.com/supabase/storage-api/issues
I don't see any such request there currently.

**Comment by rgroothuis** [comment]
Yes, correct, that is what I'm aware of indeed. Would be great if wildcards are allowed as well :-)
